cmd/init.go

```diff
@@ -84,6 +85,25 @@ var initCmd = &cobra.Command{
 
 		if len(ports) > 0 {
 			obj.Services = []Service{{PortMappings: ports}}
+		} else {
+			// Dockerfile detection
+			_, err = os.Stat("Dockerfile")
+			if err != nil {
+				fmt.Println(fileName, "is not present")
+				os.Exit(-1)
+			} else {
+				a, err := dockerfile.ParseFile("Dockerfile")
```

Comment:
the parsing should remain generic, as in whether to parse or not should be governed by other factors(like a flag or something) not on whether port info is provided.

Reply:
in this section then only extract port info